sdlib/d/gc/thread.d

```diff
@@ -181,8 +196,11 @@ public:
 	}
 
 	void stopTheWorld() shared {
+		import sdc.intrinsics;
+		auto stackTop = readFramePointer();
```

Comment:
That really is pointless. By the time the hook can do anything with the stackTop, it'll be invalid. We don't even have a scanner yet.